examples/small-molecules/README.md

```diff
@@ -3,12 +3,12 @@ A data set of small molecules selected from QM7b data set, with dft formation en
 #
 * step 1: use two sets of SOAP descriptors & print out atomic soap
 
-gen_soap_descriptors.py -fxyz small_molecules-1000.xyz -multisoap two_soap_param --peratom true
+python ../../scripts/gen_soap_descriptors.py -fxyz small_molecules-1000.xyz -multisoap two_soap_param --peratom true
 
 * step 2: visualize
 
-pca.py -fxyz ASAP-multisoap-two_soap_param.xyz -fmat MULTISOAP-two_soap_param -colors dft_formation_energy_per_atom_in_eV
```

Comment:
We decided to just use the name of the script, instead of the relative path in the examples,
because we expect people to install it

Reply:
I've fixed this now for all the example READMEs!